### ISSUE TITLE
[Request] Please, add Tabs component

### ISSUE THREAD

**Issue by WashingtonMorais** [tabler/tabler#49]
Hi...
Can you guys add a Tab Component to the template?
Tnks in advance!

**Comment by codecalm** [comment]
Hi @WashingtonMorais!
Any ideas how the tabs component should looks?

**Comment by beeman** [comment]
This might be a nice inspiration:

https://coreui.io/demo/Static_Demo/base-tabs.html

**Comment by jarboz** [comment]
Tabs are already implemented in the core of bootstrap, not sure if its included in tabler.
[https://getbootstrap.com/docs/4.0/components/navs/](url)

Explanation from getbootstrap:

> Use the tab JavaScript plugin—include it individually or through the compiled bootstrap.js file—to extend our navigational tabs and pills to create tabbable panes of local content, even via dropdown menus.

getbootstrap example:
`<nav>
  <div class="nav nav-tabs" id="nav-tab" role="tablist">
    <a class="nav-item nav-link active" id="nav-home-tab" data-toggle="tab" href="#nav-home" role="tab" aria-controls="nav-home" aria-selected="true">Home</a>
    <a class="nav-item nav-link" id="nav-profile-tab" data-toggle="tab" href="#nav-profile" role="tab" aria-controls="nav-profile" aria-selected="false">Profile</a>
    <a class="nav-item nav-link" id="nav-contact-tab" data-toggle="tab" href="#nav-contact" role="tab" aria-controls="nav-contact" aria-selected="false">Contact</a>
  </div>
</nav>
<div class="tab-content" id="nav-tabContent">
  <div class="tab-pane fade show active" id="nav-home" role="tabpanel" aria-labelledby="nav-home-tab">...</div>
  <div class="tab-pane fade" id="nav-profile" role="tabpanel" aria-labelledby="nav-profile-tab">...</div>
  <div class="tab-pane fade" id="nav-contact" role="tabpanel" aria-labelledby="nav-contact-tab">...</div>
</div>`

**Comment by WashingtonMorais** [comment]
I like @beeman  suggestion. It looks amazing!

Simple adding the bootstrap tab code works, but the style need to be changed.

![sem titulo](https://user-images.githubusercontent.com/2123599/38373975-ea898ebc-38c8-11e8-9904-d6fd46041ed0.png)

**Comment by gnpretorius** [comment]
Also worth considering the tabs within the card according to [bootstrap 4 ](https://getbootstrap.com/docs/4.1/components/card/#navigation)

If you add them right now they look as follows
![image](https://user-images.githubusercontent.com/3989178/38783028-74d6d678-40f4-11e8-99cb-972715c3afe8.png)

The above design suggestion makes sense (highlighted tab is white with background tabs transparent (for both regular tabs and tabs within cards). The tabs would also naturally sit  on top of a card (IMHO) in some cases so possibly worth considering that scenario. 

As your tab headers are white I'm not sure if the contrast is enough between selected / not selected. Happy to help take a stab at it this week and submit a PR if you have not already started.  

Great work!
Cheers

**Comment by fabiosussetto** [comment]
Using standard Bootstrap4 cart tabs:

```
<div className="card-header">
        <ul className="nav nav-tabs card-header-tabs">
          <li className="nav-item">
            <a className="nav-link active" href="#">Active</a>
          </li>
          <li className="nav-item">
            <a className="nav-link" href="#">Link</a>
          </li>
          <li className="nav-item">
            <a className="nav-link disabled" href="#">Disabled</a>
          </li>
        </ul>
      </div>
```
and overriding this bit:

```
.card-header-tabs {
  line-height: inherit;
}
```

seems enough to make the card tab look nice:

![image](https://user-images.githubusercontent.com/103664/39043666-4b08a720-448e-11e8-89d3-01c9a7da4589.png)

**Comment by ianotida** [comment]
github Tabs is much nicer than bootstrap but the same theme

![image](https://user-images.githubusercontent.com/38063617/63847145-dad1bf80-c995-11e9-84ce-ad34b590af89.png)

![image](https://user-images.githubusercontent.com/38063617/63847193-f1781680-c995-11e9-89c2-5c4aaa09d540.png)

**Comment by FreexD** [comment]
Tabs are available in @tabler/core `1.0.0-alpha.13` - https://preview.tabler.io/tabs.html . I'm closing this issue ;)
